pommerman/cli/run_battle.py

```diff
@@ -55,8 +57,10 @@ def _run(seed, record_pngs_dir=None, record_json_dir=None):
             steps += 1
             if args.render:
                 env.render(record_pngs_dir=args.record_pngs_dir,
-                           record_json_dir=args.record_json_dir,
                            mode=args.render_mode)
+            if record_json_dir is not None:
```

Comment:
why take this out of the env.render and then put it here?

Reply:
So when it runs on the server the game doesn't have to render.

Reply:
I'm a bit confused by that. The same machine that's rendering here is the one that's simulating. This isn't parallelized or being sent over a wire.

Reply:
I just thought we wouldn't be calling render when running sims on the server. I do not mind moving it back.